modules/server/src/test/scala/fs2/RPCTests.scala

```diff
@@ -80,7 +86,7 @@ class RPCTests extends RpcBaseTestSuite with BeforeAndAfterAll {
         .unsafeRunSync() shouldBe dResult33
     }
 
-    "be able to run client bidirectional streaming services" ignore {
+    "be able to run client bidirectional streaming services" in {
```

Comment:
This is related to a random failure. That's one of the reasons why your build failed: https://github.com/frees-io/freestyle-rpc/issues/164

Reply:
@juanpedromoreno thanks for pointing this out! I went ahead and re-ignored the `multiple rpc services` tests with https://github.com/frees-io/freestyle-rpc/pull/215/commits/2aaca90844031e1262690bca70341c15a7eaa28b. While the `bidirectional streaming services` tests were previously ignored, they don't seem to be failing currently. Would you like me to still set them to `ignore`, or are they ok to be left running?